### ISSUE TITLE
Inference do not support MPS

### ISSUE THREAD

**Issue by zutuanwang** [edwko/OuteTTS#51]
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
/Users/litevar/miniconda3/envs/Ali-OuteTTS/lib/python3.10/site-packages/torch/nn/functional.py:5448: UserWarning: The operator 'aten::col2im' is not currently supported on the MPS backend and will fall back to run on the CPU. This may have performance implications. (Triggered internally at /Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/mps/MPSFallback.mm:13.)
  return torch._C._nn.col2im(

**Comment by edwko** [comment]
This warning appears to come from pytorch not supporting certain operations on the mps, not the interface itself.
